gax-java/gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java

```diff
@@ -914,7 +958,10 @@ Builder setUseS2A(boolean useS2A) {
      */
     @InternalApi
     public Builder setAllowHardBoundTokenTypes(List<HardBoundTokenTypes> allowedValues) {
-      this.allowedHardBoundTokenTypes = allowedValues;
+      this.allowedHardBoundTokenTypes =
+          Preconditions.checkNotNull(
+              allowedValues, "Illegal Argument, allowedValues cannot be null");
```

Comment:
```suggestion
              allowedValues, "List of allowed HardBoundTokenTypes cannot be null");
```

Reply:
Done in https://github.com/googleapis/sdk-platform-java/pull/3591/commits/8c060dc004f45634f9ce5c0d3a3f16a8111137c7